tests/python/unittest/test_gluon.py

```diff
@@ -353,145 +240,16 @@ def test_sparse_hybrid_block():
     params['bias'] = gluon.Parameter('bias', shape=(5), dtype='float32')
     net = gluon.nn.Dense(5).share_parameters(params)
     net.initialize()
-    x = mx.nd.ones((2,5))
+    x = mx.np.ones((2,5))
     with pytest.raises(RuntimeError):
         # an exception is expected when forwarding a HybridBlock w/ sparse param
         y = net(x)
 
-def test_hybrid_block_none_args():
```

Comment:
it tests for the usage of None in block forward arguments which is a needed use case, so the test should be adapted instead of removed.